easybuild/easyblocks/b/bwa.py

```diff
@@ -61,11 +65,30 @@ def install_step(self):
         Install by copying files to install dir
         """
         srcdir = self.cfg['start_dir']
-        destdir = os.path.join(self.installdir, 'bin')
-        mkdir(destdir)
+        # copy binaries
+        bindir = os.path.join(self.installdir, 'bin')
+        mkdir(bindir)
         for filename in self.files:
             srcfile = os.path.join(srcdir, filename)
-            copy_file(srcfile, destdir)
+            copy_file(srcfile, bindir)
+
+        # copy include files
+        self.includes = glob.glob(os.path.join(srcdir, '*.h'))
```

Comment:
This is giving full paths which are then being used in the sanity check
```suggestion
        includes = glob.glob(os.path.join(srcdir, '*.h'))
        self.includes = [os.path.basename(include) for include in includes]
```